### DISCUSSION TITLE
Where can I download .osm extracts that are equivalent to .obfs from download.osmand.net?

### DISCUSSION THREAD

**Discussion by osmandlover** [osmandapp/OsmAnd#19143] [Maps Related]
Sometimes I need to swap out one of obfs available in Osmand with one that I built myself with OsmandMapCreator (OAMC). 

Right now, I grab .osm extracts from https://download.geofabrik.de/ which works, but they are different size and shape to the prebuilt obfs available at https://download.osmand.net/list . 

To avoid overlap with Osmand maps, this means that I either have to grab larger region, which takes ages to process (if my computer can even process it without crashing). Or I have to grab many small regions, which is OK but decreases performance of Osmand. 

I expect regions available on Osmand have been chosen for optimum size and shape. It would be great if I could get original .osm (or .osm.pbf or .osm.bz2) files that were used to create the OsmAnd maps, and then I could swap them out one-for-one. 

Are these osm files available anywhere? Do Osmand extract them themselves, and if so do they make available anywhere?

**Comment by vshcherb** [comment]
In https://creator.osmand.net/osm-extract/

**Comment by osmandlover** [comment]
Thanks! This is what im looking for. These files are extracted on first of each month, I understand this saves resources for the OsmAnd. if i wanted to query OSM database to get same shaped regions, how can I do that? Do you have query for each region or bounding box or similar?

**Reply by vshcherb** [reply]
Yes all the code is opensource, so you can simply search in Github and find https://github.com/osmandapp/OsmAnd-misc/tree/master/osm-planet/polygons ;-)

**Comment by osmandlover** [comment]
Hi @vshcherb , do you by any chance have an answer to this question? 

My goal is to be able to update my map with the latest data at any given time (not just the 1st of the month) with regions that are the same size and shape as the osmand ones, but with my own custom rendering_types.

I currently use download.geofabrik.de and run osmandmapcreator nightly and send the maps to my device. But my regions from geofabrik are much smaller. I think that osmand perform better when using medium sized regions like the default ones. 

I am thinking to try mirror planet.osm on my own server and extract regions of about the same size. but if the code that osmand uses to do that and the bounding polygons was available that would be very much better for me.
